.circleci/config.yml

```diff
@@ -177,9 +229,76 @@ jobs:
           path: /tmp/resource-usage
           destination: resource-usage
 
+  unit-test-windows:
+    executor:
+      name: win/default
+      shell: bash.exe
+    steps:
+      - dependencies-windows
+
+      # - run:
+      #     name: Memory Monitor
+      #     command: |
+      #       mkdir /tmp/resource-usage
+      #       export FILE=/tmp/resource-usage/memory.txt
+      #       while true; do
+      #         ps -u root eo pid,%cpu,%mem,args,uname --sort=-%mem >> $FILE
+      #         echo "----------" >> $FILE
+      #         sleep 1
+      #       done
+      #     background: true
+
+      # Note: The setup.py checksum in the cache name ensures that we retrain our expert
+      # policies, which are cached in the pytest cache, whenever setup.py changes. This
+      # is just a rough heuristic to decide when the experts should be retrained but
+      # still better than nothing.
+      - restore_cache:
+          keys:
+            - v8-win-pytest-cache-{{ checksum "setup.py" }}
+            # This prefix-matched key restores the most recent pytest cache with any
+            # checksum. If that cached policy happens to be still loadable, we avoid
+            # retraining. See https://circleci.com/docs/2.0/caching/#restoring-cache
+            - v8-win-pytest-cache-
+          paths:
+            - .pytest_cache
+
+      - run:
+          name: run tests
+          command: |
+            $ErrorActionPreference = "Stop"  # exit immediately on any error
+            .\venv\Scripts\activate
+            # Xdummy-entrypoint.py: starts an X server and sets DISPLAY, then runs wrapped command.
+            pytest -n 4 --cov=venv\Lib\site-packages\imitation `
```

Comment:
You might actually be able to use `-n auto` -- we avoided using that for Linux because the Docker executor exposed all the CPUs on the underlying machine but throttled us to a subset. I suspect this won't be an issue on Windows VM.

If we do need to specify number of CPUs manually, then consider adding a NUM_CPUS environment variable -- that way we only specify it in one place, and can update if/when we change the executor.